### DISCUSSION TITLE
Can someone help me create a .env list that scans Canadian websites for cards?

### DISCUSSION THREAD

**Discussion by SpecOpsHardman** [jef/streetmerchant#2259] [Help]
Hello,

I am looking for only an RTX 3070 or an RTX 3080 or a 6800 or 6800XT from either Best Buy, Amazon, Newegg, Memory Express, or Canada Computers in Canada. No American websites please.

Is anyone able to help me with what to write in the .env file?

I'm pretty much completely tech illiterate so I would appreciate any help!

Thank you!

**Comment by ghost** [comment]
I've responded to your request for help in discord, but you ultimately want to take a look at this page:

https://jef.codes/streetmerchant/reference/filter/

After you've installed streetmerchant, follow the instructions to create your dotenv file - then open it in a text editor (I prefer notepad++).  The site listed above will tell you which filters to apply to get the result you want - just be sure to put quotes around your selections, like this:

STORES="amazon-ca,newegg-ca"

**Comment by SpecOpsHardman** [comment]
Thank you. I am getting some error codes though. How do I fix these?

![image](https://user-images.githubusercontent.com/81671164/113198134-65396f80-921a-11eb-8e63-2a09813ae635.png)

Also eventually the program crashes after sending me to an "error" page on Amazon.

**Reply by joeish** [reply]
probably dead links in the stores ts file. You'd have to go in and manually fix the links or remove the dead ones.
